cli/src/commands/version.rs

```diff
@@ -406,20 +406,22 @@ pub async fn build_config_dependencies(
 	format: Option<&struct_fmt::Format>,
 ) -> Result<()> {
 	if let Some(matchmaker) = version.matchmaker.as_mut() {
-        // matchmaker.docker
+		// matchmaker.docker
 		let default_image_id = if let Some(docker) = matchmaker.docker.as_mut() {
-            let image_id = build_and_push_image(ctx, display_name, docker, format, None).await?;
-            docker.image_id = image_id;
-            image_id
+			let image_id = build_and_push_image(ctx, display_name, docker, format, None).await?;
+			docker.image_id = image_id;
+			image_id
 		} else {
-            None
-        };
+			None
+		};
 
-        // matchmaker.game_modes.*.docker
+		// matchmaker.game_modes.*.docker
```

Comment:
@NathanFlurry looks like CI let unformatted code through for your recent PR, I'll add a ticket to add a format checking step to CI?

Reply:
@jog1t I'll format the codebase on main and restack this so that I just see your changes and not formatting things 👍